cypress/integration/dataflow/full/running-program-spec.js

```diff
@@ -42,7 +42,7 @@ before(()=>{
 })
 
 context('Program Canvas tests',function(){
-    describe('Data Storage tests',()=>{
+    describe.skip('Data Storage tests',()=>{
```

Comment:
If the problem is just the presence of the UI alert, you may be able to fix it easily by adding a couple lines to click-through the dialog. If there's more to it than that, skipping the tests for now and circling back to them later makes sense.